### PR TITLE
More 7.2 related documentation

### PR DESCRIPTION
**Pull request by octylFractal**

### CODE REVIEW

File: source/usage/general/sessions.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    * Your selection wand binding
+    * Your navigation wand binding
+    * If CUI is currently on
+    * The last script you used
+    * Your default selector
+
+The first two are particularly important. Because of how it is stored currently, changing the default wand bindings
+in the configuration **will not** affect existing sessions. This means you must either wipe the ``sessions`` folder
+(located inside WorldEdit's configuration folder), or change it for yourself in-game using ``/tool selwand`` or
```
Comment (wizjany) [inline source/usage/general/sessions.rst:32]:
```suggestion
(located inside WorldEdit's :doc:`configuration <../../config>` folder), or change it for yourself in-game using ``/tool selwand`` or
```

File: source/usage/general/sessions.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+.. warning:: WorldEdit drops `all` sessions in single-player when leaving a world, so it is not possible to
+             directly copy between worlds or retain most tool bindings after logging out in single-player.
+
+Persistent Data
+~~~~~~~~~~~~~~~
+Some of the data in a session is `persistent`, or stored offline. This includes the following:
+
+    * Your selection wand binding
+    * Your navigation wand binding
+    * If CUI is currently on
```
Comment (wizjany) [inline source/usage/general/sessions.rst:26]:
```suggestion
    * If server-side CUI is currently on
```

**Comment by wizjany** [comment]
would add something like `(can be overridden on a per-<session> basis)` to wandItem and navigationWandItem in the config page.

**Review by wizjany — APPROVED** [review]

File: source/usage/regions/regionops.rst [hunk 3]
```diff
@@ -184,7 +184,19 @@ By default, coordinates are normalized from -1 to 1 on each axis, from the min t
...
 
 
+.. _setting-biomes:
+
 Setting Biomes
 --------------
 
 While WorldEdit mostly focuses on manipulating blocks, the ``//setbiome <biome>`` command allows you to set the biome in your selection. The biome type should be specifed by `biome id <https://minecraft.gamepedia.com/Biome#Biome_IDs>`_. If you are using a biome added by a mod, the ``namespace:`` must prefix the id, e.g. ``minecraft:plains``.
+
+.. note:: WorldEdit supports Minecraft's 3D biomes since 1.16. This means that for the most part, the biome will be set
```
Comment (me4502) [inline source/usage/regions/regionops.rst:194]: It may be worth mentioning WE version instead, as WE 7.2 supports this on 1.15

**Pull request merged**